proarc-common/src/main/java/cz/cas/lib/proarc/common/imports/TiffImporter.java

```diff
@@ -67,10 +67,10 @@
  *
  * @author Jan Pokorsky
  */
-public final class TiffImporter {
+public class TiffImporter implements ImageImporter {
```

Comment:
Logger je v nových importerech definován jako:

`Logger.getLogger(TiffAsJpegImporter.class.getName();` a
`Logger.getLogger(TiffAsJp2Importer.class.getName());`

Tedy měl jsem za to, že jsou již logovány správně, nebo takto to nestačí?